lib/index.js

```diff
@@ -458,11 +464,16 @@ module.exports = class Microinvoice {
    * @return string
    */
   prettyPrice(value) {
+    let prettifiedValue = value;
+    if (!isNaN(value)) {
```

Comment:
This isn't always the case now that I realize it, we don't necessarily want to set the value to a certain number of digits for the client of this package. 

The main use case is that for the rate / unitCost portion the units can go up to 8 digits. which this now hides. 

I think we could pass in an option which enables the user to specify the number of digits they want, and default to 2 if not set.